### ISSUE TITLE
open ssl build error on M1 Mac

### ISSUE THREAD

**Issue by sanjaneya-chwy** [depoon/SwiftLocalhost#19]
I have been using SwiftLocalHost for past few months on an intel Mac and it worked fine. My company switched to M1 Macs and I am seeing a build issue on M1 when building the project with Swiftlocalhost. Below is the error I am seeing and it looks like its originating from Criollo pod https://github.com/thecatalinstan/Criollo 

Is this a known issue and will there be support provided to use SwiftLocalHost on M1 Macs? Would be of huge help if this issue can be resolved. 

![Screen Shot 2022-03-07 at 5 08 35 PM](https://user-images.githubusercontent.com/55997758/157126785-3d2f376d-26b1-4ef7-8358-0a5e932646ba.png)

**Comment by bartekpacia** [comment]
Yeah, the problem is in the `Criollo` library that `SwiftLocalhost` uses under the hood. [Here's the related issue](https://github.com/thecatalinstan/Criollo/issues/92).

**Comment by thecatalinstan** [comment]
I've tagged a new release which should solve this.

https://github.com/thecatalinstan/Criollo/releases/tag/1.1.0

**Comment by sanjaneya-chwy** [comment]
Thanks @thecatalinstan for fixing this, helps a great deal unblock testing on the new macs. Any idea when this [PR](https://github.com/depoon/SwiftLocalhost/issues/19) will be merged so that a new release of SwiftLocalHost is made available?

**Comment by bartekpacia** [comment]
Thanks for fixing this. Unfortunately, I migrated to [another server library](https://github.com/Building42/Telegraph) before this bugfix, but much appreciated!
